stamina-core/src/main/scala/stamina/migrations.scala

```diff
@@ -36,40 +36,71 @@ package migrations {
 
   /**
    * A `Migrator[R, V]` can migrate raw values of type R from older
-   * versions to version `V` by applying a specific `Migration[R]` to it.
+   * versions to version `V` or from version one generation younger
```

Comment:
I think it's better to use the word "newer" instead of "younger" to describe Versions with greater numbers than the older ones. It's just a vocabulary clarification, as we had misunderstanding discussing this.